src/modules/esl-utils/dom/events/target.ts

```diff
@@ -9,12 +9,23 @@ export class SyntheticEventTarget implements EventTarget {
 
   private readonly _listeners: Record<string, EventListenerOrEventListenerObject[]> = {};
 
+  protected getEventListeners(): EventListenerOrEventListenerObject[];
+  protected getEventListeners(type: string): EventListenerOrEventListenerObject[];
+  protected getEventListeners(type?: string): EventListenerOrEventListenerObject[] {
+    if (!type) return Object.values(this._listeners).flat(1);
+    if (!this._listeners[type]) Object.assign(this._listeners, {[type]: []});
```

Comment:
```suggestion
    if (!this._listeners[type]) return [];
```
There is no need to mutate state, this getter should not have side effects